### ISSUE TITLE
Search Bar Generating Incorrect URLs with Duplicated Paths in Multi-Instance Setup

### ISSUE THREAD

**Issue by kmlbgn** [easyops-cn/docusaurus-search-local#442]
### Description 

I am experiencing an issue with the search functionality in Docusaurus while using it in a multi-instance setup (4 different docs instances learn, testnet, docs, ecosystem). The search bar generates incorrect URLs by duplicating the paths.

For example, instead of generating:
```
https://docs.example.com/docs/path/to/resource/#8576cd2745b74f64bd2df866b9ebc5cf
```

It generates:
```
https://docs.example.com/docs/path/to/resource//docs/path/to/resource/#8576cd2745b74f64bd2df866b9ebc5cf
```

This issue only occurs for URLs that point to a specific part of a page (using anchor links). URLs that link to a page work normally.
Taking the example below only Upgrade Module and Network Upgrades links (with page icon) will work, any others lead to 404: 

<img width="547" alt="image" src="https://github.com/user-attachments/assets/dbe7b6ae-1fde-431a-aa3f-cb08a8e510b3">

### Docusaurus config 
```

 presets: [
    [
      '@docusaurus/preset-classic',
      {
        docs: false,
        blog: false,
        theme: {
          customCss: [
            require.resolve("./css/kira.css"),
            require.resolve("./css/docu-notion-styles.css"),
          ],
        },
      },
    ],
  ],
  

(...)


  themes: [
    [
      require.resolve("@easyops-cn/docusaurus-search-local"),
      {
        // `hashed` is recommended as long-term-cache of index file is possible.
        hashed: true,
        // language: ["en", "fr"],
        indexBlog: false,
        indexPages: false,
        docsRouteBasePath: [
          "learn",
          "testnet",
          "docs",
          "ecosystem"
        ],
        searchContextByPaths: [
          { path: "learn", label: "Learn Docs" },
          { path: "testnet", label: "Testnet Docs" },
          { path: "docs", label: "Docs" },
          { path: "ecosystem", label: "Ecosystem Docs" }
        ],
      },
    ],
  ],
```

**Comment by weareoutman** [comment]
This seems to be duplicated with #408 , which is fixed since [v0.41.1](https://github.com/easyops-cn/docusaurus-search-local/releases/tag/v0.44.1).

Please run `yarn why @easyops-cn/docusaurus-search-local` or `npm ls @easyops-cn/docusaurus-search-local` to see which version you're using.

**Comment by kmlbgn** [comment]
indeed. Closing issue.
